ci-operator/step-registry/baremetalds/e2e/ovn/bgp/pre/baremetalds-e2e-ovn-bgp-pre-commands.sh

```diff
@@ -197,15 +293,33 @@ spec:
             - prefix: ${AGNHOST_SUBNET_V6}
 EOF
 
+# advertise the network
+  oc apply -f - <<EOF
+apiVersion: k8s.ovn.org/v1
+kind: RouteAdvertisements
+metadata:
+  name: ${network}
+spec:
+  networkSelector:
+    matchLabels:
+      ${label}
+  frrConfigurationSelector:
+    matchLabels:
+      ${label}
```

Comment:
I need to update the RA for CUDN in my origin PR with `frrConfigurationSelector`, once this PR is merged.